fuzzinator/ui/wui/api_handlers.py

```diff
@@ -45,7 +45,7 @@ def _convert(obj):
             except TypeError:
                 return obj
 
-        return dumps(_convert(obj))
+        return dumps(_convert(obj), indent=2, sort_keys=True)
```

Comment:
I think using tab-based indentation (`indent='\t'`) would be a better choice since it results in less bytes and we can display tabs as two spaces in an editor (if needed).

Reply:
Ok, I've changed the indent to tab.